lib/matplotlib/colors.py

```diff
@@ -1986,8 +1986,7 @@ def from_levels_and_colors(levels, colors, extend='neither'):
 
     Returns
     -------
-    (cmap, norm) : tuple containing a :class:`Colormap` and a \
-                   :class:`Normalize` instance
+    cmap, norm : `.Colormap`, `.Normalize`
```

Comment:
Multiple returns should be [separate entries](https://github.com/numpy/numpydoc/blob/master/doc/example.py#L57).